### ISSUE TITLE
A little help

### ISSUE THREAD

**Issue by MrFarhanKhan007** [JMPC14/JetpackComposeChatApplication#1]
# Hi bro, just wanted to ask one thing that, if I want to make an application similar to your using firerbase for my college project...can I do it with the free Firebase plan? free of cost without actually spending anything?....please guide me through...thanks in advance.

**Comment by mahmutconger** [comment]
Yes, it's absolutely possible; it'll make your work much easier, and if you only have one project, you can do it for free using Firebase. If you're looking to upgrade to the Blaze plan, a "pay-as-you-go" plan, don't worry. Once your credit card is charged, the recorded intervals will be displayed in this area, as no charges are made during the credit card transaction. Once entered, no payments will be made unless you exceed the specified limits. I'm confident these limits will be sufficient for your project. You can check these on the Firebase website before upgrading to the Blaze plan, or you can set your own limits by calculating your payment based on your estimated usage on the Firebase website.
